File: Models/Pari.cs
namespace SuperBowlWeb.Models
{
    public class Pari
    {
        
        public int Id { get; set; }

        public int MatchId { get; set; }
        public Jeu? Jeu { get; set; }

        public string UserId { get; set; }
        public Utilisateur? Utilisateur { get; set; }

        public int MontantMise { get; set; }
        public int? MontantGagne { get; set; }
        public DateTime DateMise { get; set; }

        public int EquipeId { get; set; }

        public Equipe? EquipeMise { get; set; }

    }
}

File: frontend/src/Modele/Pari.ts
import Equipe from "./Equipe";
import Jeu from "./Jeu";
import Utilisateur from './Utilisateur';

export default interface Pari{
    id:number|null,
    matchId:number,
    jeu:Jeu|undefined,
    userId:string,
    utilisateur:Utilisateur|undefined,
    montantMise:number,
    montantGagne:number,
    dateMise:Date,
    equipeId:number,
    equipe:Equipe|undefined
}